crates/project/src/project.rs

```diff
@@ -695,7 +695,7 @@ pub struct MarkupContent {
     pub value: String,
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
I also considered only comparing the `target` for deduping but went with this for now

Reply:
Ahh, traits.
Thank you, I would not think of this method seeing a `Vec` around.